main.tf

```
@@ -0,0 +1,19 @@
+resource "aws_account_alternate_contact" "billing" {
+  
+  alternate_contact_type = "billing"
+  count = var.aws_account_alternate_contacts.billing != null ? 1 : 0
+}
```

Couple things:

1. As a matter of convention, when used, the `count` argument should be first, and immediately below the resource/data line
2. AWS APIs are case sensitive, and the docs indicate the contact type value needs to be upper case
3. We also need to pass through the other attributes supported by the resource.
4. Let's remove the `aws_` prefix from the variable name

With that in mind, here is the suggested update to this resource. You can apply the same updates to the other two resources as well.

```suggestion
resource "aws_account_alternate_contact" "billing" {
  count = var.account_alternate_contacts.billing != null ? 1 : 0
  
  alternate_contact_type = "BILLING"
  email_address          = var.account_alternate_contacts.email_address
  name                   = var.account_alternate_contacts.name
  phone_number           = var.account_alternate_contacts.phone_number
  title                  = var.account_alternate_contacts.title
}
```